modules/indexing/src/test/java/org/apache/ignite/util/GridCommandHandlerIndexingTest.java

```diff
@@ -85,7 +90,111 @@ public void testValidateIndexesFailedOnNotIdleCluster() throws Exception {
         String out = testOut.toString();
 
         assertContains(log, out, "Index validation failed");
-        assertContains(log, out, "Checkpoint with dirty pages started! Cluster not idle!");
+        assertContains(log, out, IDLE_DATA_ALTERATION_MSG);
```

Comment:
You added tests for validate_indexes command but not for idle_verify. QA will have question about it

Reply:
@vmalin i append **additional** tests for **already**  existing functional, mentioned tests already presented here:  GridCommandHandlerTest